src/resources/TransactionTokens.ts

```diff
@@ -150,6 +151,14 @@ export interface TransactionTokenCardDetails {
     category?: CardCategory;
 }
 
+export interface TransactionTokenPaidyBilling {
+    line1?: string;
+    line2?: string;
+    city?: string;
+    state?: string;
+    zip: string;
+}
```

Comment:
No country? 🙂

Reply:
https://paidy.com/docs/en/paidycheckout.html#buyer_pay

According to the docs, not required